README.md

```diff
@@ -26,10 +26,12 @@ To get started with the quickstart application follow these steps. Steps 1-6 wil
 Under the quickstart path, run `pod install` and let the Cocoapods library create the workspace for you. Also please make sure to use **Cocoapods v1.0 and later**.
 Once Cocoapods finishes installing, open the `SwiftVoiceQuickstart.xcworkspace` and you will find a basic Swift quickstart project and a CallKit quickstart project.
 
+Note: You may need to update the [CocoaPods Master Spec Repo](https://github.com/CocoaPods/Specs) by running `pod repo update master` in order to fetch the latest specs for TwilioVoice.
+
 ### <a name="bullet2"></a>2. Create a Voice API key
 Go to the [Voice API Keys](https://www.twilio.com/console/voice/runtime/api-keys) page and create a new API key:
 
-<img src="Images/create-api-key.png"/>
+<kbd><img src="Images/create-api-key.png"/></kbd>
```

Comment:
The image changes should be part of the mute/speaker PR since this PR is just scoped to Xcode9 and cocoapods integration.